fdbclient/ClientKnobs.cpp

```diff
@@ -294,6 +294,7 @@ void ClientKnobs::initialize(Randomize randomize) {
 	init( METACLUSTER_ASSIGNMENT_FIRST_CHOICE_DELAY, 1.0 ); if ( randomize && BUGGIFY ) METACLUSTER_ASSIGNMENT_FIRST_CHOICE_DELAY = deterministicRandom()->random01() * 60;
 	init( METACLUSTER_ASSIGNMENT_AVAILABILITY_TIMEOUT, 10.0 ); if ( randomize && BUGGIFY ) METACLUSTER_ASSIGNMENT_AVAILABILITY_TIMEOUT = 1 + deterministicRandom()->random01() * 59;
 	init( TENANT_ENTRY_CACHE_LIST_REFRESH_INTERVAL,  2 ); if( randomize && BUGGIFY ) TENANT_ENTRY_CACHE_LIST_REFRESH_INTERVAL = deterministicRandom()->randomInt(1, 10);
+	init( ENABLE_CLUSTER_SHARED_STATE_MAP,         false); if( randomize && BUGGIFY ) ENABLE_CLUSTER_SHARED_STATE_MAP = true;
```

Comment:
I don't think we test this knob by randomization, because it affects the API behavior. In particular, if GRV caching is disabled, the attempt to set the transaction option USE_GRV_CACHE will fail. I would suggest writing some explicit tests for this option, in particular testing the case when clusterID is not initialzed

Reply:
Make sense. I was not aware of USE_GRV_CACHE.